http-core/src/main/scala/org/apache/pekko/http/impl/util/Rendering.scala

```diff
@@ -232,7 +233,7 @@ private[http] trait Rendering {
   /**
    * Renders the given string in double quotes.
    */
-  def ~~#!(s: String): this.type = ~~('"').putEscaped(s) ~~ '"'
+  def ~~#!(s: String): this.type = this.~~('"').putEscaped(s, Rendering.`\"`, '\\').~~('"')
```

Comment:
Another suspicious change

Reply:
This change was part of ebd17d4dce and indeed still seems required to compile

Reply:
Thanks. Not sure if it makes sense to go through every comment that I marked as suspicious, I was just concerned whether a regression was made when dealing with the merge conflicts.

If you think all of the suspicious parts I marked are legitimate just let me know and we can finally review and merge the PR.

Reply:
So far all changes seem motivated and not introduced while dealing with the merge conflicts, but I'll verify a couple more to increase our confidence they are all legitimate...